_posts/2016-09-25-Neonto-try-their-hands-at-react.mdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: keatontaylor
+title: New Policies for iOS | Webkit Blog 
+categories: link
+custom_type: link
+link_url: https://webkit.org/blog/6784/new-video-policies-for-ios/
+tags: [Design, Development, tools]
```

Comment:
Tags should be capitalized (`Tools`). I wish there was a way to programmatically do this, but I haven't figured it out yet haha